src/telemetry/telemetryController.ts

```diff
@@ -8,7 +8,8 @@ import { StorageController } from '../storage';
 const log = createLogger('analytics');
 
 export enum TelemetryEventTypes {
-  PLAYGROUND_CODE_EXECUTED = 'playground code executed'
+  PLAYGROUND_CODE_EXECUTED = 'playground code executed',
+  LINK_CLICKED = 'link clicked'
```

Comment:
I think we can make this `LINK_CLICKED` action a bit more descriptive, maybe `CREATE_ATLAS_CLUSTER_LINK_CLICKED`? - Also maybe for the action inside the webview.